tests/test_native_mcm.py

```diff
@@ -79,11 +79,18 @@ def func(x, y):
         qml.cond(m0, qml.RY)(y, wires=1)
         return qml.classical_shadow(wires=0)
 
-    with pytest.raises(
-        DeviceError,
-        match=f"not accepted with finite shots on lightning.qubit",
-    ):
-        func(*params)
+    if device_name == "lightning.qubit":
+        with pytest.raises(
+            DeviceError,
+            match=f"not accepted with finite shots on lightning.qubit",
+        ):
+            func(*params)
+    else:
```

Comment:
why do we have different errors for LQ and LK?

Reply:
That's an old/new API difference.